USAGE.md

```diff
@@ -0,0 +1,33 @@
+# How to use Testing Farm as GitHub Action in your projects
+
+Before start of using Testing Farm as GitHub Action in your projects,
+there are several steps which you have to do it.
+
+## As Testing Farm team for public or private ranch
+
+Before any implementation as Testing Farm team for API_KEY here [onboarding](https://docs.testing-farm.io/general/0.1/onboarding.html).
+
+## Setup TMT plan in GitHub / GitLab repos
+
+In your GitHub or GitLab repos create:
+* [.fmf](https://github.com/sclorg/sclorg-testing-farm/tree/main/.fmf) directory with file [version](https://github.com/sclorg/sclorg-testing-farm/blob/main/.fmf/version) which contains 1.
+* [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) directory which will contain TMT plans for Testing Farm
+* create a TMT plan like it is already done here [c9s](https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf)
+* The TMT plan contains instructions how to setup, install and execute tests on quest provided by Testing Farm. For more information see [documentation](https://tmt.readthedocs.io/en/stable/examples.html#plans) or take an inspiration from our [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) diretory
+
+## Setup GitHub Action
+
+As soon as you have your TMT plan ready then let's create an GitHub Actions for your use case
+
+### Testing by GitHub Actions
+
+Create a directory `.github/workflows` in your GitHub repository.
+
+### Testing in pull request
```

Comment:
The wording "Testing in pull request" didn't make sense  to me until I read the description. So, it's technically correct, but maybe we can rewrite it to something more clear, like "Configuration for tests to be run on pull-requests"... or something similar.

Reply:
The new caption is `### Testing pull request changes via Testing Farm`